<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientHistory</class>
 <widget class="QDialog" name="PatientHistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Patient History</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="formTitle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Patient History</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget {
	background:#ffffff;
	border-radius:9px;
	border:1px solid #bbbbbb;
}

QDateEdit, QTimeEdit {
	background:#ffffff;
}

QPlainTextEdit {
	background:#ffffff;
	border:1px solid #dddddd;
	padding:2px;
}

#buttonBox QAbstractButton {
	background:#ffffff;
	border-radius:9px;
	border:1px solid #bbbbbb;
	padding:5px;
	padding-left:10px;
	padding-right:10px;
	margin:2px;
}

#buttonBox QAbstractButton:pressed {
	background: #444444;
	color:#ffffff;
}

QScrollBar {
	background:none;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="margin">
       <number>5</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>695</width>
             <height>656</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">#scrollAreaWidgetContents {
	background:#ffffff;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="margin">
             <number>1</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QGroupBox" name="groupBox">
                <property name="styleSheet">
                 <string notr="true">#groupBox {
	background: #ffffff;
}

QLabel {
	background: #ffffff;
}</string>
                </property>
                <property name="title">
                 <string>Personal History</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_6">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                     <item>
                      <widget class="QLabel" name="label">
                       <property name="text">
                        <string>Surgeries:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_5">
                       <property name="text">
                        <string>Allergies:</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                     <item>
                      <widget class="QPlainTextEdit" name="history_surgeries">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>70</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPlainTextEdit" name="history_allergies">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>70</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                     <item>
                      <widget class="QLabel" name="label_2">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="font">
                        <font>
                         <weight>50</weight>
                         <bold>false</bold>
                        </font>
                       </property>
                       <property name="text">
                        <string>Diseases:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_6">
                       <property name="text">
                        <string>Other:</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_5">
                     <item>
                      <widget class="QPlainTextEdit" name="history_diseases">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>70</height>
                        </size>
                       </property>
                       <property name="font">
                        <font>
                         <pointsize>10</pointsize>
                        </font>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPlainTextEdit" name="history_other2">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>70</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_3">
                <property name="styleSheet">
                 <string notr="true">#groupBox_3 {
	background: #ffffff;
}

QLabel {
	background: #f7f7f7;
}</string>
                </property>
                <property name="title">
                 <string>Family History</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_11">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_10">
                   <item>
                    <widget class="QPlainTextEdit" name="history_family"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_5">
                <property name="styleSheet">
                 <string notr="true">#groupBox_5 {
	background: #ffffff;
}

QLabel {
	background: #ffffff;
}

QComboBox, QComboBox QWidget  {
	background: none;
}</string>
                </property>
                <property name="title">
                 <string>Habits</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_15">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_14">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_6">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QLabel" name="label_3">
                       <property name="styleSheet">
                        <string notr="true">padding-right:3px;
</string>
                       </property>
                       <property name="text">
                        <string>Smoking:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QComboBox" name="history_habit_smoking">
                       <item>
                        <property name="text">
                         <string>No</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>Occasionaly</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>Regularly</string>
                        </property>
                       </item>
                      </widget>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>2</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_4">
                       <property name="styleSheet">
                        <string notr="true">padding-right:3px;
</string>
                       </property>
                       <property name="text">
                        <string>Sports:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QComboBox" name="history_habit_sports">
                       <item>
                        <property name="text">
                         <string>No</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>Occasionaly</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>Amateur</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>Professional</string>
                        </property>
                       </item>
                      </widget>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer_2">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>2</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_7">
                       <property name="styleSheet">
                        <string notr="true">padding-right:3px;</string>
                       </property>
                       <property name="text">
                        <string>Alcohol:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QComboBox" name="history_habit_alcohol">
                       <item>
                        <property name="text">
                         <string>No</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>Occasionaly</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>Regularly</string>
                        </property>
                       </item>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_8">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>Other:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPlainTextEdit" name="history_habit_other">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>40</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_2">
                <property name="styleSheet">
                 <string notr="true">#groupBox_2 {
	background: #ffffff;
}

QLabel {
	background: #f7f7f7;
}</string>
                </property>
                <property name="title">
                 <string>Medications</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_9">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_8">
                   <item>
                    <widget class="QPlainTextEdit" name="history_medications">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>60</height>
                      </size>
                     </property>
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_4">
                <property name="styleSheet">
                 <string notr="true">#groupBox_4 {
	background: #ffffff;
}

QLabel {
	background: #f7f7f7;
}</string>
                </property>
                <property name="title">
                 <string>Other</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_13">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_12">
                   <item>
                    <widget class="QPlainTextEdit" name="history_other">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="font">
                      <font>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PatientHistory</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PatientHistory</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
